tests/features/python/requirements

```diff
@@ -0,0 +1,6 @@
+pytest
+allure
```

Comment:
Should this be `allure-pytest` ? Couldn't seem to install `allure` on python3.9 by directly doing `pip3 install allure` but may be missing  

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/8011761/168480508-9094ce0b-b3d5-4f4f-ba1e-22613df2747a.png">

Reply:
Ah, yes, you right, sry
Should be allure-pytest